spec/fabricators/user_fabricator.rb

```diff
@@ -0,0 +1,5 @@
+Fabricator(:user) do
+  email { Faker::Internet.email }
+  full_name { Faker::Name.name }
+  password 'password'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.